draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -260,7 +266,7 @@ public void minimize() {
    */
   public void closeToRight() {
     if (viewDragHelper.smoothSlideViewTo(dragView, transformer.getOriginalWidth(),
-        getHeight() - transformer.getMinHeightPlusMargin())) {
+            getHeight() - transformer.getMinHeightPlusMargin())) {
```

Comment:
All this extra whitespace you have here and in the rest of the file will be gone if you format the project using the correct checkstyle.